docs/DocumentationStyleGuide/language-grammar/articles.rst

```diff
@@ -0,0 +1,20 @@
+.. _language--grammar-ii-articles:
+
+Language & Grammar II: Articles
+===============================
+
+Using articles is always preferred in your documentation piece as it
```

Comment:
This section might need one or more examples so it's more accessible to ESOL readers.